modules/virt-configuring-ip-for-vm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-configuring-ip-for-vm_{context}"]
+= Configuring an IP address for a virtual machine
+
+You can use cloud-init to configure an IP address for a virtual machine. The IP address can be provisioned to be dynamic or static. 
+
+You can add or edit the cloud-init configuration of an existing virtual machine configuration. If you edit a running virtual machine, the virtual machine must be rebooted for the changes to take effect.
```

Comment:
cloud-init is applied only on the first boot. the configuration cannot be edited

Reply:
Ah, I must have misunderstood you earlier. 
That will have a knock-on effect for an earlier module.